src/sql/SqlResult.ts

```diff
@@ -99,23 +97,19 @@ export interface SqlResult extends AsyncIterable<SqlRowType> {
     /**
      * Returns row metadata of the result.
      * @returns SQL row metadata if rows exist in the result; otherwise null.
-     * If SQL execution was not successful, this promise is rejected with an error.
      */
     getRowMetadata(): Promise<SqlRowMetadata | null>;
 
     /**
      * Return whether this result has rows to iterate. False if update count is returned, true if rows are returned.
-     * If SQL execution was not successful, this promise is rejected with an error.
      */
     isRowSet(): Promise<boolean>;
 
     /**
-     * Returns the number of rows updated by the statement or -1 if this result is a row set.
-     * If SQL execution was not successful, this promise is rejected with an error.
-     * @returns Update count
+     * Returns the number of rows updated by the statement or `-1` if this result is a row set.
+     * @returns update count
```

Comment:
We still have a mixin of documentation types for the return values (see this method and isRowSet). It would be good to be consistent